backend/UnicornRewards.API/Services/IContactService.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace UnicornRewards.API.Services
+{
+    public interface IContactService<T>
+    {
+        Task<T> AddContact(T id);
```

Comment:
If this will be an async method it needs to be named AddContactAsync

Reply:
Name Changed, change updated on "fixes to contact list functionality" commit